packages/server/README.md

```diff
@@ -78,18 +78,49 @@ Allow `Forwarded`, `X-Forwarded-Proto`, and `X-Forwarded-Host` headers to overri
 Additional headers sent in `Access-Control-Allow-Headers` (`string[]`).
 
 #### `options.generateUrl`
-Control how the upload url is generated (`(req, { proto, host, baseUrl, path, id }) => string)`)
+
+Control how the upload URL is generated (`(req, { proto, host, path, id }) => string)`)
+
+This only changes the upload URL (`Location` header).
```

Comment:
Can we add an exemplary use case when generateUrl is useful? For example, to rewrite the URL so that it fits the format that is expect by a proxy for incoming requests?

Reply:
I did add an example for adding a prefix based on the user from the request. Unless we know a common format change for a proxy, I think the result would also just be another psuedo-code example with some non existing functions to call.

Reply:
I wasn't thinking about code here, but an example situation explained in prosa where `generateUrl` is necessary or helpful. This makes it easier for users to understand when they would want to change the upload URL.